components/team_sidebar/team_sidebar_controller.jsx

```diff
@@ -139,6 +139,7 @@ export default class TeamSidebar extends React.PureComponent {
                 <div
                     className='team-wrapper'
                     id='teamSidebarWrapper'
+                    role='navigation'
```

Comment:
Can we remove the role here in favour of `sidebar.jsx`?

Reply:
that will meant that this part is visible and not under any role and `axe` will return an error
![Screenshot from 2019-12-20 20-03-37](https://user-images.githubusercontent.com/6043510/71270932-d88bda80-2363-11ea-9aa8-5f6640f22400.png)
What am I doing wrong?

Reply:
Does having `role='navigation'` on both `sidebar` and `team_sidebar_controller` work?

Reply:
It's like this now.` components/sidebar/sidebar.jsx ` and `components/team_sidebar/team_sidebar_controller.jsx ` have `role='navigation'`. There is no error like this from axe

Reply:
Okay, we're probably covered then.